packages/crew_sudo.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Crew_sudo < Package
+  description 'Workaround for using sudo on ChromeOS crosh shell (ChromeOS v117+)'
+  homepage 'https://github.com/supechicken/crew-sudo'
+  version '1.0'
+  license 'GPL-3'
+  compatibility 'all'
+
+  source_url 'https://github.com/supechicken/crew-sudo.git'
+  git_hashtag '1.0'
+
+  no_compile_needed
+
+  def self.install
```

Comment:
Could we offload the install logic to the actual package itself? It feels more appropriate there...

Reply:
The logic here just copying the files, it sounds better to me to leave it here (an installation script will be needed if we move this logic to the `crew-sudo` repo)

Reply:
That's kind of the idea... 😅

If we have the choice (which we do), I'd prefer that we offload as much logic to packages as possible.

Reply:
Change made. Now the install logic is located at [crew-sudo/install.sh](https://github.com/supechicken/crew-sudo/blob/v1.1/install.sh)